### ISSUE TITLE
Visit data history displayed differently in different devices

### ISSUE THREAD

**Issue by ellyasafaa** [onaio/rdt-standard#830]
The sync process has gone well, I didn't record it separately, but it worked well. However, when checking on the history visit, it displayed in a different way.
This video represented the 1st device:
https://drive.google.com/file/d/11X7GEiGAwJCooTLh0Mmk9apBijQag3--/view?usp=sharing

This video represented the 2nd device:
https://drive.google.com/file/d/1EQCEf4xoCbWbCtGOhw19kagQKywxpa9V/view?usp=sharing

**Comment by vincent-karuri** [comment]
@syedowaisali90 could you check on the date used in creating the visits. Verify that it is the date the event is originally created on the client and not on the date of creation on the server.

**Comment by ellyasafaa** [comment]
Now, the history is not displayed at all:
https://drive.google.com/file/d/1HlRAp1OL8FrHA7TN8ZyuFDMl2klFO0_S/view?usp=sharing

**Comment by syedowaisali90** [comment]
> @syedowaisali90 could you check on the date used in creating the visits. Verify that it is the date the event is originally created on the client and not the date of creation on the server.

@vincent-karuri I have checked the date event date is originally created date on the client-side and maybe, it's a syncing issue

**Comment by syedowaisali90** [comment]
> Now, the history is not displayed at all:
> https://drive.google.com/file/d/1HlRAp1OL8FrHA7TN8ZyuFDMl2klFO0_S/view?usp=sharing

@ellyasafaa is it was reproducing or it happened only once.

**Comment by vincent-karuri** [comment]
Thanks @syedowaisali90 . 

@ellyasafaa could you provide exact steps to reproduce this?

**Comment by ellyasafaa** [comment]
> > Now, the history is not displayed at all:
> > https://drive.google.com/file/d/1HlRAp1OL8FrHA7TN8ZyuFDMl2klFO0_S/view?usp=sharing
> 
> @ellyasafaa is it was reproducing or it happened only once.

I tried it twice, but for the first one, I didn't take any video of it. Now, I'm checking it, I'm still not able to reproduce that, but the strange thing is in Ujian Satu patient history, I've just checked the history and it appeared. So, I thought, the problem is in the sync process. However, when I checked it, the history that appeared in Ujian Satu's history on March 8, was the history of Andika that I recently created. You can match the info of Ujian Satu's history here: https://drive.google.com/file/d/1HlRAp1OL8FrHA7TN8ZyuFDMl2klFO0_S/view?usp=sharing. It doesn't match. And I have created Andika's history and it matched. You can directly check on the user of rarang_lotim1 or any Lombok Timur district user.

**Comment by ellyasafaa** [comment]
So the Ujian Satu's history on March 8 is not recorded, but it turned out putting other patient's history in the history section

**Comment by ellyasafaa** [comment]
> Thanks @syedowaisali90 .
> 
> @ellyasafaa could you provide exact steps to reproduce this?

As in the video I sent you, I filled in the history as usual: choosing one patient > clicking symptom and history form > filling in all information > submit, and it just happened even after I synced it

**Comment by ellyasafaa** [comment]
For this comment https://github.com/onaio/rdt-standard/issues/830#issuecomment-797187373, as a user, I think the issue is still related to each other: patient history issue in general. However, if you think you'll need to open a new issue on Github to fix this, just let me know.

**Comment by vincent-karuri** [comment]
thanks @ellyasafaa. @syedowaisali90 could you try creating a new patient and creating visit data to see if it shows. And create a second patient and record data against them as well. See if any of the issues arise?

**Comment by syedowaisali90** [comment]
@vincent-karuri I have created a patient named `Jimmy` and submitted the visit and checked that the visit which was showing in the patient history and it was showing correctly and done the same process with creating another patient named `Jack`
